docs/application/get-started/iot.md

```diff
@@ -2,15 +2,28 @@
 
 The Get Started Tutorial teaches you how to create basic Tizen applications for IoT devices.
 
+## Hardware Setup for IoT Development
+
+To start developing Tizen IoT applications, follow the [Get Started guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi 3 or 4 board.
+
 ## .NET Application
 
 The following guide demonstrates how to create and run a basic Tizen .NET application, which displays some text on the screen with no user interaction:
 
--   [IoT .NET application](../dotnet/get-started/iot/first-app.md)
+- [Create Your First Tizen IoT .NET Application](../dotnet/get-started/iot/first-app.md)
+
+	An IoT application is targeted to run on an IoT device. The applications can take advantage of a wide variety of functionalities available on IoT devices.
 
-    An IoT application is targeted to run on an IoT device. The applications can take advantage of a wide variety of functionalities available on IoT devices.
-    
 First, ensure that you have installed the latest version of Visual Studio Tools for Tizen. [**Learn more >**](../vstools/install.md)
 
-## Hardware Setup for IoT Development
-To start developing Tizen IoT applications, follow the [Get Started guide](http://tizenschool.org/tutorial/191/) that demonstrates how to set up the Raspberry Pi 3 or 4 board.
+## Native Service Application
+
+The following guide demonstrates how to create and run a basic Tizen IoT native service application, which displays some text on the log with no user interaction:
+
+- [Create Your First Tizen IoT Native Service Application](..\native\get-started\iot\first-app.md)
+
+    An IoT native service application can be created using the C language and is targeted to run on Raspberry Pi. The application uses the native APIs, which provide various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
```

Comment:
How about following the similar list item name with other get-started pages?

Also, the link is broken.

```suggestion
You can create rich applications in C to run on Raspberry Pi with various interfaces to the device hardware.

To begin your Tizen Native Service application development journey, use the following quick start guides:

- [IoT Native Service application](../native/get-started/iot/first-app.md)
```

Reply:
Please change guides to guide:

To begin your Tizen Native Service application development journey, use the following quick start guide:

Reply:
I just tried to write it in a format similar to the .NET Application above.
@mijin85cho  What you suggest loos better, I think. Thank you. I will apply them.
@surbhig12 I will change to guide. Thanks.